test/Unit/models/Import/Service/ResultImporterTest.php

```diff
@@ -294,7 +348,51 @@ function ($callItemId, $responseId) {
             );
 
         $this->expectException(ImportResultException::class);
-        $this->expectExceptionMessage('SCORE_TOTAL_MAX cannot be higher than 1, 1.1 provided');
+        $this->expectExceptionMessage('SCORE_TOTAL cannot be higher than SCORE_TOTAL_MAX:1, 1.1 provided');
+
+        $this->sut->importByResultInput($this->input);
+    }
+
+    public function testCreateByImportResultCanWorkWithScoreTotalMaxEmpty(): void
+    {
+        $this->input->addOutcome('item-1', 'SCORE', 1.1);
+
+        $this->resultStorage
+            ->expects($this->once())
+            ->method('getDeliveryVariables')
+            ->willReturn(
+                [
+                    777 => [
+                        (object)[
+                            'variable' => $this->createTestVariable(1, 'SCORE_TOTAL'),
+                        ],
+                    ],
+                    999 => [
+                        (object)[
+                            'variable' => $this->createTestVariable(0, 'SCORE_TOTAL_MAX'),
```

Comment:
I do not expect that it does not exist, but if it is empty, so that after [cast to float here](https://github.com/oat-sa/extension-tao-outcome/blob/6380fd7285525968326020b7c5ee00ed6344876b/models/Import/Service/ResultImporter.php#L237) it become 0, so I mocked such scenario to test [this](https://github.com/oat-sa/extension-tao-outcome/pull/243/files#diff-f9a6969df492261e45e23aa01ed33aac1efef38d0150096c37d423e6ffe650b4R100) `!empty($scoreTotalMax) &&`

Reply:
@gabrielfs7 a new test case above is covering it `testCreateByImportResultCanWorkWithScoreTotalMissing` both are missing there.